docs/API/components/form-builder.md

```diff
@@ -1,21 +1,80 @@
 # Fliplet Form Builder JS APIs
 
+## Retrieve an instance
+
+### `Fliplet.FormBuilder.get()`
+
+Retries the first or a specific form chartInstances
+
 ```js
 // the "get()" accepts the form name in case you have multiple forms on the page
-Fliplet.FormBuilder.get().then(function (form) {
```

Comment:
I'd keep an example without passing `name`. Also, instad of `name` use `'foo'` as a value